CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 1.1.1
```

Comment:
This is not really a "bug fix / patch" version but an additive version (new feature) so I think we should bump to 1.2.0 instead.